.github/workflows/tauri-release.yml

```
@@ -17,45 +17,154 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: Tauri Build Artifacts (${{ matrix.os }})
           path: |
             src-tauri/target/release/bundle
+
+  create-release:
+    runs-on: ubuntu-latest
+    needs: build-tauri
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)